cumulus/parachains/runtimes/starters/seedling/src/lib.rs

```diff
@@ -75,7 +75,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("seedling"),
 	impl_name: create_runtime_str!("seedling"),
 	authoring_version: 1,
-	spec_version: 10000,
+	spec_version: 1,
```

Comment:
you're saying this won't be a problem here?

Reply:
This is more like a template runtime.